asynction/server.py

```diff
@@ -176,24 +172,33 @@ def from_spec(
             if url_parse_result.path:
                 kwargs["path"] = url_parse_result.path
 
+            server_security = server.security
+
         asio = cls(spec, validation, docs, app, **kwargs)
-        asio._register_handlers(default_error_handler)
+        asio._register_handlers(default_error_handler, server_security=server_security)
         return asio
 
     def _register_namespace_handlers(
         self,
         namespace: str,
         channel_handlers: ChannelHandlers,
         channel_bindings: Optional[ChannelBindings],
+        server_security: Optional[Sequence[SecurityRequirement]] = None,
     ) -> None:
+        on_connect = None
```

Comment:
```suggestion
        on_connect = lambda: *args, **kwargs = None
```
Instead of injecting a string reference to a callable that we would need to import within the spec data.